axelrod/strategy_transformers.py

```diff
@@ -9,9 +9,12 @@
 import copy
 import inspect
 import random
+from typing import Any
```

Comment:
Please reorder the imports (alphabetical, std lib, then 3rd party, then project specific)